_cite/plugins/google-scholar.py

```diff
@@ -10,7 +10,7 @@ def main(entry):
     """
 
     # get api key
-    api_key = os.environ.get("GOOGLE_SCHOLAR_API_KEY", "")
+    api_key = get_safe(os.environ, "GOOGLE_SCHOLAR_API_KEY", "")
```

Comment:
Yeah this is tricky, because it's Serp API but no one knows what that is. I made it GOOGLE_SCHOLAR_API_KEY to match with the plugin name and service name.

I'll add a little comment to hopefully add clarity.